tcms/core/tests/test_report_tag.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+import unittest
+from tcms.core.templatetags.report_tags import percentage
+
+
+class TestReportTagPercentage(unittest.TestCase):
+
+    def test_percentage_calculation(self):
+
+        self.assertEqual(percentage(5.0, 2.5), '200.0%')
+        with self.assertRaises(ValueError):
+            percentage('five', 'ten')
+        self.assertEqual('0.0%', percentage(0, 2.0))
+        self.assertEqual('0%', percentage(2.0, 0))
+        self.assertEqual(percentage(1, 1), '100.0%')
+        self.assertEqual(percentage(1, 3), '33.3%')
```

Comment:
The last 2 scenarios represent the following scenarios;
1) 0 digits after the decimal divider
2) 2 or more digits after the decimal divider

Missing is a scenario which results in a percentage value with only 1 digit after the decimal divider.

Reply:
Still not resolved.